src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -113,6 +115,8 @@ public SwerveModule(MotorConfig driveMotorConfig, MotorConfig turningMotorConfig
         actualSpeedLog = new DoubleLogEntry(datalog, "/swerve/" + ID +"/setSpeed"); //Logs actual speed in meters per second
         desiredAngleLog = new DoubleLogEntry(datalog, "/swerve/" + ID +"/setAngle"); //Logs desired angle in radians
         actualAngleLog = new DoubleLogEntry(datalog, "/swerve/" + ID +"/actualAngle"); //Logs actual relative angle in radians
+
+        if(errorCode != 0){DriverStation.reportError("An Error has occured in SwerveModule", null);}
```

Comment:
Reporting that an error occurred in SwerveModule is overly-broad
Checking for errors at different points would allow for more specific/helpful logging messages